web/models/survey.ex

```diff
@@ -648,4 +648,16 @@ defmodule Ask.Survey do
   def delete_multi(survey) do
     Multi.delete(Multi.new, :survey, survey)
   end
+
+  def first_window_started_at(%{started_at: nil} = _survey), do: nil
+
+  def first_window_started_at(%{schedule: schedule, started_at: started_at} = _survey) do
+    calculated_started_at = Schedule.next_available_date_time(schedule, started_at)
```

Comment:
I suppose that the `Schedule.next_available_date_time` will always return the same given the same schedule and date, right?

Also, a survey's schedule never changes?

Is there a reason for calculating this every time instead of storing it as field in the Survey table? 

I'm wondering this ☝️ thinking of reports that may be extracted from the db. Are there such about a survey? is this field necessary?

Reply:
> is this field necessary?

Yes, it is. I wasn't aware!

Changing it! Thanks!!! 🙌